tower-balance/src/pool.rs

```diff
@@ -71,14 +73,17 @@ where
                     return Ok(Async::NotReady);
                 }
 
+                tracing::trace!("decided to add service to loaded pool");
```

Comment:
TIOLI: should we also record the number of services here?
```suggestion
                tracing::trace!(pool.services = self.services, message = "decided to add service to loaded pool");
```